modules/rosa-sdpolicy-networking.adoc

```diff
@@ -0,0 +1,65 @@
+- External master load balancer for Red Hat that is reserved for cluster management by Red Hat. Access is strictly controlled, and communication is only possible from whitelisted bastion hosts.
+- Default external router/ingress load balancer that is the default application load balancer, denoted by apps in the URL. The default load balancer can be configured in OCM to be either publicly accessible over the Internet or only privately accessible over a pre-existing private connection. All application routes on the cluster are exposed on this default router load balancer, including cluster services such as the logging UI, metrics API, and registry.
+- Optional: secondary router/ingress load balancer that is a secondary application load balancer, denoted by apps2 in the URL. The secondary load balancer can be configured in OCM to be either publicly accessible over the Internet or only privately accessible over a pre-existing private connection. If a `Label match` is configured for this router load balancer, then only application routes matching this label are exposed on this router load balancer, otherwise all application routes are also exposed on this router load balancer.
+- Optional: Load balancers for Services enable non-HTTP/SNI traffic and non-standard ports for services.  These load balancers can be mapped to a service running on {product-title} to enable advanced ingress features, such as non-HTTP/SNI traffic or the use of non-standard ports. These can be purchased in groups of 4 for standard clusters or can be provisioned without charge in CCS clusters, however each AWS account has a quota which link:https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-limits.html[limits the number of Classic Load Balancers] that can be used within each cluster.
+
+== Cluster ingress
+Project administrators can add route annotations for many different purposes, including ingress control through IP whitelisting.
+
+Ingress policies can also be changed by using NetworkPolicy objects, which leverage the `ovs-networkpolicy` plugin. This allows for full control over the ingress network policy down to the pod level, including between pods on the same cluster and even in the same namespace.
+
+All cluster ingress traffic will go through the defined load balancers. Direct access to all nodes is blocked by cloud configuration.
+
+== Cluster egress
+Pod egress traffic control through `EgressNetworkPolicy` objects can be used to prevent or limit outbound traffic in {product-title}.
+
+Public outbound traffic from the master and infrastructure nodes is required and necessary to maintain cluster image security and cluster monitoring. This requires the 0.0.0.0/0 route to belong only to the internet gateway, it is not possible to route this range over private connections.
```

Comment:
s/the 0.0.0.0/0 route/the `0.0.0.0/0` route

Reply:
s/internet/Internet
s/gateway, it is/gateway; it is

Reply:
Done.